__tests__/endpoints.test.js

```diff
@@ -237,4 +238,74 @@ describe('GET /api/articles/:article_id/comments', () => {
         expect(response.body.msg).toBe('article does not exist');
         });
     });
-});
\ No newline at end of file
+});
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('responds with a 201 status code', () => {
+        const newComment = {
+            author: "icellusedkars",
+            body: "I carry a frog — yes."
+        };
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(201)
+    });
+    test('should insert a new comment to the db and sends the new comment back to the client', () => {
+        const newComment = {
+            author: "icellusedkars",
+            body: "I carry a frog — yes."
+        };
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+        //const date = new Date()
+            expect(typeof response.body.comment_id).toBe('number')
+            expect(typeof response.body.article_id).toBe('number')
+            expect(typeof response.body.author).toBe('string')
+            expect(typeof response.body.body).toBe('string')
+            expect(typeof response.body.votes).toBe('number')
+            expect(typeof response.body.created_at).toBe('string')
+           /* I originally tested for the exact values but I was getting a discrepancy
+           between my timestamps. Unsure how to resolve this but wanted to mention it as
+           i think it's the right test to do
+           ({
+                comment_id: 19,
+                article_id: 1,
+                author: "icellusedkars",
+                body: "I carry a frog — yes.",
+                votes: 0,
+                created_at: date.toJSON()
```

Comment:
we can just copy the exact value of the created at and put it here, or we can have the exact values of the other properties and have an expect.any(String) for the created at